code/script.js

```diff
@@ -30,13 +59,157 @@ const showMessage = (message, sender) => {
+  setTimeout(() => prepareFood(), 2000)
+})
+}
+
+
+// Fråga 5 Sammanfatta (två parametrar) amount och selected
+const askForConfirmation = (amount, selectedDish) => {
+inputWrapper.innerHTML =''
+showMessage(`Alright, we have recived your order for ${amount} ${selectedDish}! Please confirm your order`)
+inputWrapper.innerHTML = `
+<button id='restart'> Nej✋ </button>
+<button id='confirm'> Yes👍 </button>
+`
+document.getElementById('restart').addEventListener('click', () => location.reload())
+document.getElementById('confirm').addEventListener('click', () => {
+  inputWrapper.innerHTML = ''
+  showMessage('Yes🥳', 'user')
+  setTimeout(() => showMessage (`Thank you ${userName} for your order of ${amount} ${selectedDish}. Do you want fries for free🤠?`, 'bot'), 1000)
+  setTimeout(() => askForFries(), 4000)
+})
+}
+
+
+//Fråga 4
+const askForAmount = selectedDish => {
+  inputWrapper.innerHTML =''
+  showMessage(`How many ${selectedDish} do you want?`, 'bot')
+  inputWrapper.innerHTML = `
+  <input type="number" id="amount"/>
+  <button id="amount-btn" class="send-btn">
+  Skicka
+  </button>
+  `
+  const amountInput = document.getElementById('amount')
+
+  document.getElementById('amount-btn').addEventListener('click', () => {
+    showMessage(amountInput.value, 'user')
+    setTimeout(() => askForConfirmation(amountInput.value, selectedDish), 1000)
+  })
+}
+
+
+//Fråga 3 Vilken mat användaren vill ha
+//show message funktionen, 
+
+const askForDish = foodChoice => {
+  
+  showMessage(`What kind of ${foodChoice} do you want?`, 'bot')
+
+  if (foodChoice === 'original') {
+  //Orginal Burger choices In drop down
+  inputWrapper.innerHTML = veggieBurgerMenu
+
+  } else if (foodChoice ==='cheese'){
+  //visa burger meny
+  inputWrapper.innerHTML = cheeseBurgerMenu
+
+  } else {
+  //visa burger meny
+  inputWrapper.innerHTML = chilliBurgerMenu
+  }
+
+  const selectedDish = document.getElementById('select')
+  selectedDish.addEventListener('change', () =>{
+    showMessage(selectedDish.value, 'user')
+    setTimeout(() => {
+    inputWrapper.innerHTML =''
+    askForAmount(selectedDish.value)
+    }, 1000)
+  })
+}
+
+
+//Fråga 2 Svarar usern med valuet från deras input, som sedan van välja vad hen vill beställa
+//Besvarar användaren med function och message. 
+//Lägger till HTML element för val i knappar. 
+//lägger till klick event för meny val som skickar vidare till nästa funktion "askForDish"
+const askForFood = (userName) =>{
+  showMessage(`Hi ${userName}👋 Nice to meet you! What would you like to order?`,'bot')
+  inputWrapper.innerHTML=`
+  <button id="originalButton">Original Burger</button>
+  <button id="cheeseButton">Cheese Burger</button>
+  <button id="chilliButton">Chilli Burger</button>
+  `
+  
+  document
+  .getElementById("originalButton")
+  .addEventListener('click', () => 
+  {
+    showMessage(`I want a original burger`, 'user')
```

Comment:
If you're not using JavaScript inside the string - you could use regular quotation marks instead of template literals :)